File: src/LightNap.Core/StaticContents/Dto/Response/PublishedStaticContentDto.cs
using LightNap.Core.StaticContents.Enums;

namespace LightNap.Core.StaticContents.Dto.Response
{
    /// <summary>
    /// An end-user-ready CMS item.
    /// </summary>
    public class PublishedStaticContentDto
    {
        /// <summary>
        /// The content.
        /// </summary>
        public required string Content { get; set; }

        /// <summary>
        /// The format of the content.
        /// </summary>
        public required StaticContentFormat Format { get; set; }
    }
}

File: src/lightnap-ng/src/app/core/backend-api/dtos/static-contents/response/published-static-content-dto.ts
import { StaticContentFormat } from "@core/backend-api/static-content-format";

/**
 * Static content for rendering.
 */
export interface PublishedStaticContentDto {
    /**
     * The content.
     */
    content: string;

    /**
     * The format of the content.
     */
    format: StaticContentFormat;
}
